### DISCUSSION TITLE
No HDR

### DISCUSSION THREAD

**Discussion by Cepheus** [aidanmacgregor/Netflix_ATV_L3_DRM_Uncertified_Mod#22] [General]
I am very pleased to finally have a way to have Netflix working on my Formovie Theater 4K.
A shame that it seems HDR content is not available in HDR, only HD.

Is that a normal behavior or can I do something to fix that?
Cheers

**Comment by aidanmacgregor** [comment]
Sorry this is expected, Netflix will only send HDR content encrypted with L1 DRM
